### PR TITLE
config(metrics): tag task duration metric with task type

### PR DESCRIPTION
**Pull request by dreynaud**
With this, we should be able to see a breakdown of time taken for different task types (looking at you, `ServerGroupCacheForceRefreshTask`)

### CODE REVIEW

**Comment by ajordens** [comment]
This may (or may not) blow out the tag space and prevent us from looking at multiple hours of this metric?

**Comment by robzienert** [comment]
I said this, but Daniel looked at me like I was being ridiculous.

File: orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandler.kt [hunk 1]
```diff
@@ -113,24 +113,25 @@ class RunTaskHandler(
...
       }
     }
   }
 
-  private fun trackResult(stage: Stage, taskModel: com.netflix.spinnaker.orca.pipeline.model.Task, status: ExecutionStatus) {
+  private fun trackResult(stage: Stage, task: Task, taskModel: com.netflix.spinnaker.orca.pipeline.model.Task, status: ExecutionStatus) {
     val id = registry.createId("task.invocations")
       .withTag("status", status.toString())
       .withTag("executionType", stage.execution.type.name.capitalize())
+      .withTag("taskType", task::class.qualifiedName)
```
Comment (robfletcher) [inline orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandler.kt]: You already have access to `taskModel` which has an `implementingClass` property that gives you exactly this value. No need to introduce another parameter.

Comment (dreynaud) [reply orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandler.kt]: sweet, thanks!

**Comment by dreynaud** [comment]
@ajordens I count <300 task classes. This is moderately high, but I don't think it is prohibitively expensive. There are lots of metrics with a lot more cardinality, so I'm not overly worried about this one.

**Pull request merged**